### ISSUE TITLE
Accuracy is reduced after 5 epoch

### ISSUE THREAD

**Issue by yintao-he** [vinsis/ternary-quantization#3]
Hey, do you try to train lots of epoches? I found that the accuracy will be reduced after 5 epoch, until to 10%. It really confused me, could u please give me a hand?

**Comment by vinsis** [comment]
No I did not train for many epochs. The network converges within a couple of epochs. Maybe you need to decrease the learning rate gradually.
